src/v2/Components/NavBar/Menus/DropDownMenu/DropDownMenu.tsx

```diff
@@ -5,10 +5,37 @@ import React from "react"
 import styled from "styled-components"
 import { DropDownSection } from "./DropDownSection"
 import { Menu, MenuItem } from "v2/Components/Menu"
+import { MenuData } from "../../menuData"
+
+const VIEW_ALL_TOP_MARGINS = {
+  HeaderArtworksDropdown: "120px",
+  HeaderArtistsDropdown: "80px",
+} as const
+
+const LinkMenuItem = styled(MenuItem).attrs({
+  px: 1,
+  py: 0.5,
+  color: "black60",
+  variant: "text",
+  hasLighterTextColor: true,
+})``
+
+LinkMenuItem.displayName = "LinkMenuItem"
+
+const ViewAllMenuItem = styled(LinkMenuItem)`
+  text-decoration: underline;
+  color: ${color("black100")};
+`
+
+ViewAllMenuItem.displayName = "ViewAllMenuItem"
+
+const Links = styled(Box)`
+  border-right: 1px solid ${color("black10")};
+`
 
 interface DropDownNavMenuProps {
   width?: string
-  menu: any
+  menu: MenuData
```

Comment:
I think the only reason this was `any`'d was to get around the type guard thing.